survey-sdk/src/test/java/co/nimblehq/surveysdk/serializer/SerializerTest.kt

```diff
@@ -0,0 +1,117 @@
+package co.nimblehq.surveysdk.serializer
+
+
+import co.nimblehq.surveysdk.entity.SurveyDetailEntity
+import co.nimblehq.surveysdk.entity.SurveyListEntity
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+
+@RunWith(MockitoJUnitRunner::class)
+class SerializerTest {
+
+    lateinit var moshi: Moshi
+
+    @Before
+    fun setUp() {
+        moshi = Moshi.Builder()
+            .add(KotlinJsonAdapterFactory())
+            .build()
+    }
+
+
```

Comment:
redundant break line

Please be more careful with the code we write. Sometimes, just some small formatting issue but it would take our feeling a little bit down when checking the code. I believe you also feel good and comfortable when checking a smooth code also 🤓

Reply:
Agree with you @luongvo

Reply:
Updated